src/tools/pub/cmd/pub-downgrade.md

```diff
@@ -99,14 +99,23 @@ trying to find a set of versions that work with your package from what's already
 available.
 
 
-## Options {#options}
 
-The `pub downgrade` command supports the
-[`pub get` options](/tools/pub/cmd/pub-get#options).
+## Options
 
 For options that apply to all pub commands, see
 [Global options](/tools/pub/cmd#global-options).
 
+`--offline`
+: Use locally cached packages instead of accessing the network.
+
+`--no-offline`
+: Connect to the network to check for hosted dependencies. This is the 
+default setting.
```

Comment:
I think that we can be more terse when it comes to expressing what the default setting is. How about: _Connect to the network to check for hosted dependencies (default)._

This is often what tools do. @kwalrath, thoughts?

Reply:
It might be clearer to have the "default" at the beginning of the explanation, or after the period. Maybe:

```
:  Default. Connect ...
```

But if we combine negatable flags, that won't work. Maybe, in that case:

```
`--[no-]offline`
: By default, pub connects to the network... (`--offline`). To use cached packages instead, use `--no-offline`.
```

Reply:
(It took me a couple of times to get the sense of the flag right... I think.)